content/blog/2017/2017-02-02-scm-api-2-take2.adoc

```diff
@@ -0,0 +1,116 @@
+---
+:layout: post
+:title: SCM API 2.0 Release Take 2
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+In January we link:https://jenkins.io/blog/2017/01/17/scm-api-2/[announced the release of SCM API 2.0].
+There were four important issues identified after the original release was published and in order to allow those issues to be resolved, the updates were pulled from the update center.
+
+These issues were:
+
+link:https://issues.jenkins-ci.org/browse/JENKINS-41121[JENKINS-41121]::
+Migration of GitHub branches from 1.x to 2.x resulted in a change of branch identity which, now that other bugs in Branch API had been fixed, resulted in a rebuild of all GitHub Branches (not PRs) after an upgrade to GitHub Branch Source Plugin 2.0.0.
```

Comment:
what is "branch identity"? This is one long run-on sentence. Can you make it a little more clear?